README.md

```diff
@@ -71,14 +54,8 @@ Use [this walkthrough](https://ibm.github.io/type/code.html) to learn the founda
 
 ## Performance
 
-There are two distinct capabilities of this codebase and you are welcome to only import what you need:
-1. Core - All capabilities not listed below.
-2. Styles - Provides a map of the raw type scale and then a map and mixins of the type sets.
-
-|                          | Includes core font references? | Includes scale and styles? | Includes line lengths? | File Size | Minified | Gzipped  |
-|--------------------------|---------------------|-----------------------|-------------------------|-----------|----------|-------|
-| IBM Type                 | Yes                 | Yes                   | Yes                     | 39kb      | 34kb      | 2.6kb |
-| _styles.scss               | Yes                 | No                    | No                      | 15kb       | 11kb      | 1.3kb |
-| _core.scss               | Yes                 | No                    | No                      | 24kb       | 23kb      | 1.3kb |
+| ibm-type.css | ibm-type.min.css | Gzipped  |
```

Comment:
Should we provide any breakdowns for individual font families? Might be easy to auto-generate this information tbh.

Reply:
Yeah, but I think it would be confusing until #15 is knocked out